prime-dotnet-webapi/prime.csproj

```diff
@@ -30,13 +30,15 @@
     <PackageReference Include="SoapCore" Version="1.1.0.2" />
     <PackageReference Include="Swashbuckle.AspNetCore" Version="5.0.0" />
     <PackageReference Include="Swashbuckle.AspNetCore.Newtonsoft" Version="5.0.0" />
-    <PackageReference Include="CompareNETObjects" Version="4.73.0" />
+    <PackageReference Include="CompareNETObjects" Version="4.65.0" />
     <PackageReference Include="Microsoft.Extensions.Http" Version="3.1.4" />
     <PackageReference Include="Wkhtmltopdf.NetCore" Version="3.0.0" />
     <PackageReference Include="IdentityModel" Version="4.3.0" />
     <PackageReference Include="LinqKit.Microsoft.EntityFrameworkCore" Version="3.0.0" />
     <PackageReference Include="DelegateDecompiler.EntityFrameworkCore" Version="0.28.0" />
     <PackageReference Include="Flurl" Version="3.0.1" />
+    <PackageReference Include="RazorEngine.NetCore" Version="3.1.0" />
+    <PackageReference Include="Microsoft.CodeAnalysis.CSharp.Workspaces" Version="3.6.0" />
```

Comment:
What is this package reference for?

Reply:
The razor engine need a version defined or it get dependency loading errors when you restore